### PR TITLE
Exclude tests from bpr following :defaults of :files

### PR DESCRIPTION
**Pull request by jian-lin**
cc package author @iliabv 

ref: https://github.com/NixOS/nixpkgs/issues/335442

### CODE REVIEW

**Review by iliabv — APPROVED** [review]

**Comment by riscy** [comment]
Thanks both.  Before I hit merge, I should note that if you rename this excluded file to bpr-tests.el or create a test subdirectory instead, then this recipe update won't be required.  I'm happy to merge if this isn't straightforward - but the default recipe has a few small conventions in it like this that can be useful to rely on without having to make new PRs to MELPA.

Since I'm here checking, I can also point out a few salient issues:

- 183:9: error: You should depend on (emacs "24.1") if you need `process-live-p'.
- 194:11: error: You should depend on (emacs "24.1") if you need `process-live-p'.
- 254:7: error: You should depend on (emacs "24.1") if you need `scroll-down-command'.
- bpr.el needs *formal* license boilerplate and/or an [SPDX-License-Identifier](https://spdx.dev/ids/)

**Comment by riscy** [comment]
@iliabv Thoughts on the previous comment?

**Comment by riscy** [comment]
I'll merge this for now.

**Pull request merged**
